dockertest/subtest.py

```diff
@@ -291,6 +296,22 @@ def make_subsubtest_config(self, all_configs, parent_config,
         self.config = self.make_config(all_configs, parent_config,
                                        self.config_section)
 
+    def require_docker_version(self, wanted):
```

Comment:
Thought: since it's just that one config option, probably no harm in just adding a parameter for it.